templates/docs/base/forms.md

```diff
@@ -64,6 +64,16 @@ To use checkbox component in headings use the `.p-checkbox--heading` variant of
 View example of the heading checkbox components
 </a></div>
 
+#### Tri-state checkbox
+
+<span class="p-label--new">New</span>
+
+When a checkbox requires a state between checked and unchecked, the HTML attribute `aria-checked` can be set to `mixed`. In JavaScript, `checkbox.indeterminate = true;` will also cause the third state to appear, and should be set along with the `aria-checked` attribute.
```

Comment:
As far as I understand `aria-checked` is meant to be used on elements that are not native checkbox input.
Native inputs, but default implement accessibility features, so don't require additional aria.

In this case, native checkbox with `indeterminate` set to true, doesn't seem to require also setting `aria-checked`.
In same way, you don't set `aria-checked="true"` when setting `checkbox.checked = true`.

see: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

I think we should encouraging using native `checkbox.indeterminate`, and only using `aria-checked` in cases when JS is not available.

Reply:
Didn't we agree to use `aria-checked` for the sake of Percy? Without this, we rely on JavaScript to get the indeterminate state, so we won't see the impact in Percy.

Reply:
@sowasred2012 Yes, what I mean is that we shouldn't document it as a default way to get this state (using JS to set `checkbox.intederminate = true` should be default. And if it's used `aria-checked` is not needed, so we shouldn't say that it "should be set along with the `aria-checked`".

So, I think in the docs we should say that to get intederminate state JS should be used. In cases when it can't `aria-checked="mixed"` can be used instead